enatega-multivendor-app/App.js

```
@@ -147,11 +131,12 @@ export default function App() {
 
   // For Sentry
   useEffect(() => {
-    if (SENTRY_DSN) {
+    // if (SENTRY_DSN) {
+    if (false) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)